pom.xml

```diff
@@ -86,9 +86,16 @@
       <dependency>
         <groupId>com.sonatype.nexus</groupId>
         <artifactId>nexus-platform-api</artifactId>
-        <version>3.3.20180828-152410.8e581c6</version>
+        <version>3.3-SNAPSHOT</version>
```

Comment:
I don't know if we want to do a version bump on nexus-platform-api to 3.4 yet or not.

In general the practice for nexus-platform-api is to do a version bump when starting new work after a release, just in case we had to do a fix release of some kind. I might check with @whyjustin to see what he'd recommend for these changes because I'm not certain if the fix you're making would warrant a new minor version.